tt-train/.github/workflows/builld_and_test_all.yaml

```diff
@@ -0,0 +1,97 @@
+name: Build Tests
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  merge_group:
+
+jobs:
+  run-tests:
+    runs-on: ["n150"]
```

Comment:
our labels our "N150"
I'm not sure if this will run with "n150"

Reply:
It worked fine btw.